firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/impl/TesterApiHttpClientTest.java

```diff
@@ -251,28 +251,33 @@ public void makePostRequest_whenConnectionFails_throwsError() throws Exception {
   @Test
   public void makeUploadRequest_writesRequestBodyAndSetsCorrectHeaders() throws Exception {
     String responseJson = readTestFile("testSimpleResponse.json");
-    InputStream responseInputStream = new ByteArrayInputStream(responseJson.getBytes(UTF_8));
-    when(mockHttpsURLConnection.getResponseCode()).thenReturn(200);
-    when(mockHttpsURLConnection.getInputStream()).thenReturn(responseInputStream);
-    ByteArrayOutputStream requestBodyOutputStream = new ByteArrayOutputStream();
-    when(mockHttpsURLConnection.getOutputStream()).thenReturn(requestBodyOutputStream);
-
-    ContentResolver contentResolver =
-        ApplicationProvider.getApplicationContext().getContentResolver();
-    ShadowContentResolver shadowContentResolver = shadowOf(contentResolver);
-    ByteArrayInputStream inputStream = new ByteArrayInputStream("Test post body".getBytes(UTF_8));
-    Uri uri = Uri.parse("file:///path/to/data");
-    shadowContentResolver.registerInputStream(uri, inputStream);
 
-    testerApiHttpClient.makeUploadRequest(TAG, TEST_PATH, TEST_AUTH_TOKEN, uri);
-
-    assertThat(new String(requestBodyOutputStream.toByteArray(), UTF_8))
-        .isEqualTo("Test post body");
-    verify(mockHttpsURLConnection).setDoOutput(true);
-    verify(mockHttpsURLConnection).setRequestMethod("POST");
-    verify(mockHttpsURLConnection).addRequestProperty("Content-Type", "application/json");
-    verify(mockHttpsURLConnection).addRequestProperty("X-Goog-Upload-Protocol", "raw");
-    verify(mockHttpsURLConnection).addRequestProperty("X-Goog-Upload-File-Name", "screenshot.png");
-    verify(mockHttpsURLConnection).disconnect();
+    // Setup test streams. They should be closed but close them anyway just to be safe.
```

Comment:
I think the comment is missing some words. Maybe entirely drop the comment entirely or explain that since these are in-memory streams they just get garbage collected and don't strictly need to be closed to avoid resource leaks.

Reply:
Got it. Removed the comment.